assets/dfn-panel.css

```diff
@@ -53,21 +53,33 @@ dfn {
   color: #333;
 }
 
+.dfn-panel > div {
+  display: flex;
+  gap: 0.5em;
+}
+
 .dfn-panel a.self-link {
   font-weight: bold;
+  margin-right: auto;
 }
 
-.dfn-panel .dfn-exported {
-  float: right;
+.dfn-panel .marker {
```

Comment:
when the panel is at the top of the viewport, the IDL button is not clickable as the "ReSpec" command menu overlaps it (at least in my arguably quick & specific test); I'm not sure how to address this cleanly